pkg/scheduler/backend/cache/cache.go

```diff
@@ -757,4 +757,12 @@ func (cache *cacheImpl) updateMetrics() {
 	metrics.CacheSize.WithLabelValues("assumed_pods").Set(float64(len(cache.assumedPods)))
 	metrics.CacheSize.WithLabelValues("pods").Set(float64(len(cache.podStates)))
 	metrics.CacheSize.WithLabelValues("nodes").Set(float64(len(cache.nodes)))
+
+	// we intentionally keep them with the deprecation and will remove at v1.33.
+	// nolint:staticcheck
```

Comment:
```suggestion
	//nolint:staticcheck
```

> Use //nolint instead of // nolint because machine-readable comments should have no space by Go convention.
https://golangci-lint.run/usage/false-positives/

Reply:
I know `// nolint` (with space) also works, but I mean it's not recommended. I remember there was a discussion on golangci-lint side about it.